SimulationPool.java

```diff
@@ -67,11 +55,7 @@ public SimulationPool(int games, double[][] weightSets, boolean isBatch, boolean
        	//For each generation, we run a 1000 games. For each game, we assign a thread.
         for(int i=0; i< games; i++){
             Simulation game;
-            if (isBatch) {
-                game = new Simulation(weightSets[i], findAverage, numRepetitions);
-            } else {
-                game = new Simulation(weightSets[0]);
-            }
+            game = new Simulation(h, weightSets[i], numRepetitions);
```

Comment:
I infer from this is that batch processing is now the default and fixed?

Reply:
Yup! It was too messy so I just took the other one out